app/code/Magento/SimpleProductGraphQl/Model/Resolver/Cart/AddSimpleProductsToCart.php

```diff
@@ -0,0 +1,213 @@
+        $this->userContext = $userContext;
+        $this->arrayManager = $arrayManager;
+        $this->productRepository = $productRepository;
+        $this->cartHydrator = $cartHydrator;
+        $this->guestCartRepository = $guestCartRepository;
+        $this->dataObjectFactory = $dataObjectFactory;
+        $this->cartRepository = $cartRepository;
+        $this->maskedQuoteIdToQuoteId = $maskedQuoteIdToQuoteId;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null) : Value
+    {
+        $cartHash = $this->arrayManager->get('input/cart_id', $args);
+        $cartItems = $this->arrayManager->get('input/cartItems', $args);
+
+        if (!isset($cartHash)) {
+            throw new GraphQlInputException(
+                __('Missing key %1 in cart data', ['cart_id'])
+            );
+        }
+
+        if (!isset($cartItems)) {
+            throw new GraphQlInputException(
+                __('Missing key %1 in cart data', ['cartItems'])
+            );
+        }
+
+        $cart = $this->getCart((string) $cartHash);
+
+        foreach ($cartItems as $cartItem) {
+            $sku = $this->arrayManager->get('details/sku', $cartItem);
+            $product  = $this->productRepository->get($sku);
+
+            $cart->addProduct($product, $this->getBuyRequest($cartItem));
+
+            if ($cart->getData('has_error')) {
+                throw new GraphQlInputException(__('Cart has an error: %1 (%2)', $this->getCartErrors($cart), $sku));
+            }
+        }
+
+        $this->cartRepository->save($cart);
+
+        $result = function () use ($cart) {
+            return [
+                'cart' => $this->cartHydrator->hydrate($cart)
+            ];
+        };
+
+        return $this->valueFactory->create($result);
+    }
+
+    /**
+     * @param array $cartItem
+     *
+     * @return DataObject
+     */
+    private function getBuyRequest($cartItem): DataObject
+    {
+        $qty = $this->arrayManager->get('details/qty', $cartItem);
+        $customizableOptions = $this->arrayManager->get('customizable_options', $cartItem, []);
+        $customOptions = [];
+
+        foreach ($customizableOptions as $customizableOption) {
+            $customOptions[$customizableOption['id']] = $customizableOption['value'];
+        }
+
+        return $this->dataObjectFactory->create([
+            'data' => [
+                'qty' => $qty,
+                'options' => $customOptions
+            ]
+        ]);
+    }
+
+    /**
+     * @param CartInterface|Quote $cart
+     *
+     * @return string
+     */
+    private function getCartErrors($cart): string
+    {
+        $errorMessages = [];
+
+        /** @var AbstractMessage $error */
+        foreach ($cart->getErrors() as $error) {
+            $errorMessages[] = $error->getText();
+        }
+
+        return implode(PHP_EOL, $errorMessages);
```

Comment:
It is probably better to use semicolon or another separator for the messages in GraphQL output.